back/engines/commercial/analysis/spec/acceptance/taggings_spec.rb

```diff
@@ -68,4 +68,29 @@
       expect { Analysis::Tagging.find(id) }.to raise_error(ActiveRecord::RecordNotFound)
     end
   end
+
+  post 'web_api/v1/analyses/:analysis_id/taggings/bulk_create' do
+    parameter :tag_id, 'The ID of the tag to associate the input with', required: true
+    parameter :filters, 'The filters to select the inputs to associate the tag with', required: true
+
+    ValidationErrorHelper.new.error_fields(self, Analysis::Tagging)
+
+    let(:analysis) { create(:analysis) }
+    let(:analysis_id) { analysis.id }
+    let!(:inputs) { create_list(:idea, 3, project: analysis.project) }
```

Comment:
I think the list helper is creating the ideas with the same title somehow so I created an extra idea to use the search on below.